client/overview/inbox-notifications/index.js

```diff
@@ -0,0 +1,315 @@
+/* eslint-disable camelcase */
```

Comment:
It seems that a large percentage of js files in this repo make use of the above `eslint-disable camelcase`, maybe we should just disable it? https://github.com/Automattic/woocommerce-payments/blob/develop/.eslintrc#L32

Reply:
Did you ever follow up on this? I thought I remembered seeing something about this elsewhere?

Reply:
I had left a comment in Slack, but didn't get a huge response there, aside from one person.
So I didn't change anything "yet"